jbpm-wb-forms/jbpm-wb-forms-client/src/test/java/org/jbpm/workbench/forms/client/display/KieWorkbenchFormsStartProcessDisplayerTest.java

```diff
@@ -0,0 +1,97 @@
+import org.jbpm.workbench.forms.display.api.ProcessDisplayerConfig;
+import org.jbpm.workbench.forms.display.service.KieWorkbenchFormsEntryPoint;
+import org.jbpm.workbench.pr.model.ProcessDefinitionKey;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kie.workbench.common.forms.dynamic.client.DynamicFormRenderer;
+import org.kie.workbench.common.forms.dynamic.service.shared.impl.MapModelRenderingContext;
+import org.mockito.Mock;
+import org.uberfire.mvp.Command;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(GwtMockitoTestRunner.class)
+public class KieWorkbenchFormsStartProcessDisplayerTest {
+
+    @Mock
+    protected DynamicFormRenderer formRenderer;
+
+    @Mock
+    protected Caller<KieWorkbenchFormsEntryPoint> service;
+
+    @Mock
+    protected KieWorkbenchFormsEntryPoint kieWorkbenchFormsEntryPoint;
+
+    @Mock
+    protected KieWorkbenchFormRenderingSettings formRenderingSettings;
+
+    @Mock
+    protected Command onClose;
+
+    @Mock
+    protected Command onRefresh;
+
+    protected ProcessDisplayerConfig processDisplayerConfig;
+
+    protected KieWorkbenchFormsStartProcessDisplayer kieWorkbenchFormsStartProcessDisplayer;
+
+    @Before
+    public void init() {
+        ProcessDefinitionKey processDefinitionKey = new ProcessDefinitionKey( "test-serverTemplateId",
+                                                                              "test-deploymentId",
+                                                                              "test-processId",
+                                                                              "test-processDefName");
+        processDisplayerConfig = new ProcessDisplayerConfig(processDefinitionKey,
+                                                            "test");
+        processDisplayerConfig.setRenderingSettings(formRenderingSettings);
+
+        kieWorkbenchFormsStartProcessDisplayer = new KieWorkbenchFormsStartProcessDisplayer(formRenderer, service);;
+        when(formRenderer.isValid()).thenReturn(true);
+        when(formRenderingSettings.getTimestamp()).thenReturn(100000l);
+
+        MapModelRenderingContext mapModelRenderingContext = mock(MapModelRenderingContext.class);
+        when(formRenderingSettings.getRenderingContext()).thenReturn(mapModelRenderingContext);
+        when(mapModelRenderingContext.getModel()).thenReturn(null);
+        when(service.call(any())).thenReturn(kieWorkbenchFormsEntryPoint);
+    }
+
+    @Test
+    public void testStartProcessFromDisplayer(){
+
+        kieWorkbenchFormsStartProcessDisplayer.initConfigs(processDisplayerConfig, onClose, onRefresh);
+        kieWorkbenchFormsStartProcessDisplayer.startProcessFromDisplayer();
+
+        verify(service).call(any());
+        verify(kieWorkbenchFormsEntryPoint).startProcessFromRenderContext(any(),
```

Comment:
would be good if you match the expected values in here. I would recommend bringing some of the values you defined in init to here. Those are execution/parameter specific, init should generally be a setup only.